tests/comments.test.js

```diff
@@ -0,0 +1,186 @@
+            idDiscussion: "616b0efeba862c9a697da9db"
+        };
+        chai.request(server).post("/comments").send(comment)
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(201);
+            response.body.should.be.a("object");
+            response.body.should.have.property("_id");
+            response.body.should.have.property("comment");
+            response.body.should.have.property("idAccount");
+            response.body.should.have.property("idDiscussion");
+            response.body.should.have.property("dateCreation");
+            idComment = response.body._id;
+            done();
+        });
+    });
+
+    it("POST /Comments Bad Request Account", (done) =>{
+        const comment = {
+            comment: "Creo que los cuentos son un poco largos", 
+            idAccount: "6168cf9563929f8f000c7616", 
+            idDiscussion: "616b0efeba862c9a697da9db"
+        };
+        chai.request(server).post("/comments").send(comment)
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La cuenta no existe");
+            done();
+        });
+    });
+
+    it("POST /Comments Bad Request Disccussion", (done) =>{
+        const comment = {
+            comment: "Creo que los cuentos son un poco largos", 
+            idAccount: "6168cf9563929f8f000c7614", 
+            idDiscussion: "616b0efeba862c9a697da9dc"
+        };
+        chai.request(server).post("/comments").send(comment)
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La discusion no existe");
+            done();
+        });
+    });
+
+    it("POST /Comments Bad Request", (done) =>{
+        const comment = {
+            comment: "  ", 
+            idAccount: "6168cf94", 
+            idDiscussion: "616b0efeba86"
+        };
+        chai.request(server).post("/comments").send(comment)
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+});
+
+describe("DELETE Tests Comments",()=>{
+    before((done) =>{
+        const login = {
+            username: "MiroStar",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("DELETE /Comments", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("DELETE /Comments", (done) => {
```